target/linux/ar71xx/files/arch/mips/ath79/mach-xd6700.c

```diff
@@ -0,0 +1,109 @@
+/*
+ *  Yuncore XD6700 (from Atheros AP147) reference board support
+ *
+ *  Copyright (C) 2014 Matthias Schiffer <mschiffer@universe-factory.net>
+ *  Copyright (C) 2015 Sven Eckelmann <sven@open-mesh.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License version 2 as published
+ *  by the Free Software Foundation.
+ */
+
+#include <linux/phy.h>
+#include <linux/gpio.h>
+#include <linux/platform_device.h>
+#include <linux/ar8216_platform.h>
+
+#include <asm/mach-ath79/ar71xx_regs.h>
+#include <asm/mach-ath79/ath79.h>
+
+#include "common.h"
+#include "dev-ap9x-pci.h"
+#include "dev-eth.h"
+#include "dev-gpio-buttons.h"
+#include "dev-leds-gpio.h"
+#include "dev-m25p80.h"
+#include "dev-usb.h"
+#include "dev-wmac.h"
+#include "machtypes.h"
+#include "pci.h"
+
+#define XD6700_GPIO_LED_SYSTEM     1
+
+#define XD6700_GPIO_LED_GREEN      12
+#define XD6700_GPIO_LED_RED        13
+
+#define XD6700_GPIO_BTN_RESET      17
+
+#define XD6700_KEYS_POLL_INTERVAL  20     /* msecs */
+#define XD6700_KEYS_DEBOUNCE_INTERVAL     (3 * XD6700_KEYS_POLL_INTERVAL)
+
+#define XD6700_MAC0_OFFSET         0x1000
```

Comment:
This is probably wrong, please confirm it. In most cases, eth MACs are stored at the beginning of the ART partition. At offset `0x1000` starts ART data (and WLAN MAC isn't stored at offset `0x` in ART!).

Reply:
MAC0 offset is the WLAN MAC in the art, eth MAC's are at the beginning of the ART as per the code.